### DISCUSSION TITLE
Confidence in ultralytics prediction

### DISCUSSION THREAD

**Discussion by Simon334** [ultralytics/ultralytics#14726] [General]
Hello, I would like to ask how to add the ability to display only those bounding boxes that have a confidence factor above 0.45 to the image display using Ultralytics prediction. Here is part of my code:
if img is not None:
                        results = model(img)
                        for result in results:
                            boxes_yolo = result.boxes
                            masks = result.masks
                            keypoints = result.keypoints
                            probs = result.probs
                        result.show()

**Comment by pderrenger** [comment]
@Simon334 to display only bounding boxes with a confidence factor above 0.45, you can set the `conf` parameter when calling the `predict` method. Modify your code to include `conf=0.45` in the `model` call. This will filter out detections with confidence below 0.45. If you encounter any issues, ensure you are using the latest version of the Ultralytics package.
